src/main.js

```diff
@@ -3,13 +3,19 @@ import FiltersView from './view/filters.js';
 import { render, RenderPosition } from './framework/render.js';
 import TripInfoView from './view/trip-info.js';
 import { PointModel } from './model/point-model.js';
+import { filter } from './util.js';
+import { generateFilter } from './mock/filter.js';
 
 const tripMainContainer = document.querySelector('.trip-main');
 const tripEventsContainer = document.querySelector('.trip-events');
 const pointModal = new PointModel();
```

Comment:
```suggestion
const pointModеl = new PointModel();
```
поправить дальше в местах использования